core/include/earcut.hpp

```diff
@@ -1 +1 @@
-Subproject commit 1384c97a79c322e1aad37fd317bf87be5dbb24fa
+Subproject commit 195b2fdae9a354c0f86592f320df4988c896f822
```

Comment:
This code deviated a bit from mapbox/master - so I can't tell. The nullptr deref is also probably only shown as defect since coverity didn't do the math for not hitting sum != 0